core/src/components/toggle/toggle.tsx

```diff
@@ -137,10 +137,11 @@ export class Toggle implements ComponentInterface {
   @Event() ionStyle!: EventEmitter<StyleEventDetail>;
 
   @Watch('disabled')
-  disabledChanged() {
+  protected disabledChanged() {
     this.emitStyle();
-    if (this.gesture) {
```

Comment:
Do we need this fix here? I thought toggle doesn't currently receive hover/active states from item.

Reply:
🤔 doesn't look like it. I could of sworn I tested each of these in the browser first to figure out which ones needed to be adjusted and the hover styles were applying, but maybe I was mistaken with another component. 

I'll revert all the changes to this file.